### PR TITLE
Improve acceptance tests

### PR DESCRIPTION
**Pull request by henryrecker-pingidentity**
Update acceptance tests to follow the pattern of minimal -> maximal -> import -> minimal. This helps ensure we are covering all attributes and covers removing attributes to go back to minimal configuration.

### CODE REVIEW

File: internal/resource/config/oauth/authserversettings/scopes/commonscope/oauth_auth_server_settings_scopes_common_scope_resource.go [hunk 1]
```diff
@@ -19,31 +19,31 @@ import (
...
 	Name        types.String `tfsdk:"name"`
 	Description types.String `tfsdk:"description"`
 	Dynamic     types.Bool   `tfsdk:"dynamic"`
 }
 
 // GetSchema defines the schema for the resource.
-func (r *oauthAuthServerSettingsScopesCommonScopesResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+func (r *oauthAuthServerSettingsScopesCommonScopeResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
 	schema := schema.Schema{
 		Description: "Manages a OauthAuthServerSettingsScopesCommonScopes.",
```
Comment (wesleymccollam) [inline internal/resource/config/oauth/authserversettings/scopes/commonscope/oauth_auth_server_settings_scopes_common_scope_resource.go:48]: Looks like this needs to be something like "Manages an Oauth Auth Server Settings Scopes Common Scope"

File: internal/resource/config/oauth/authserversettings/scopes/exclusivescope/oauth_auth_server_settings_scopes_exclusive_scope_resource.go [hunk 2]
```diff
@@ -19,33 +19,33 @@ import (
...
 	Description types.String `tfsdk:"description"`
 	Dynamic     types.Bool   `tfsdk:"dynamic"`
 }
 
 // GetSchema defines the schema for the resource.
-func (r *oauthAuthServerSettingsScopesExclusiveScopesResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+func (r *oauthAuthServerSettingsScopesExclusiveScopeResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
 	schema := schema.Schema{
-		Description: "Manages a OauthAuthServerSettingsScopesExclusiveScopes.",
+		Description: "Manages a OauthAuthServerSettingsScopesExclusiveScope.",
```
Comment (wesleymccollam) [inline internal/resource/config/oauth/authserversettings/scopes/exclusivescope/oauth_auth_server_settings_scopes_exclusive_scope_resource.go:48]: Same thing here => Looks like this needs to be something like "Manages an Oauth Auth Server Settings Scopes Exclusive Scope"

File: internal/acctest/keypairsigningimport/key_pair_signing_import_test.go [hunk 3]
```diff
@@ -35,6 +35,12 @@ func TestAccKeyPairsSigningImport(t *testing.T) {
 		format:   format,
 		password: password,
 	}
+	updatedResourceModel := keyPairsSigningImportResourceModel{
```
Comment (wesleymccollam) [inline internal/acctest/keypairsigningimport/key_pair_signing_import_test.go:38]: very nice

File: internal/acctest/keypairsslserverimport/key_pair_ssl_server_import_test.go [hunk 4]
```diff
@@ -34,6 +34,12 @@ func TestAccKeyPairsSslServerImport(t *testing.T) {
 		format:   format,
 		password: password,
 	}
+	updatedResourceModel := keyPairsSslServerImportResourceModel{
```
Comment (wesleymccollam) [inline internal/acctest/keypairsslserverimport/key_pair_ssl_server_import_test.go:37]: I like these updates for certs

**Review by wesleymccollam — CHANGES_REQUESTED** [review]

File: internal/resource/config/oauth/authserversettings/scopes/commonscope/oauth_auth_server_settings_scopes_common_scope_resource.go (hunk 1 above)
Comment (henryrecker-pingidentity) [reply internal/resource/config/oauth/authserversettings/scopes/commonscope/oauth_auth_server_settings_scopes_common_scope_resource.go]: Updated both

**Review by wesleymccollam — APPROVED** [review]

**Pull request merged**